[2/n] Remove references to TorchScript in PyTorch docs

yiming0416 opened the pull request:
Summary: Removed jit_language_reference.md

Test Plan:
CI

Rollback Plan:

Differential Revision: D78308133

svekars reviewed (approved):
LGTM, please double-check in the preview.

yiming0416 committed:
[2/n] Remove references to TorchScript in PyTorch docs (#158306)

Summary:
Pull Request resolved: https://github.com/pytorch/pytorch/pull/158306

Removed jit_language_reference.md

Test Plan:
CI

Rollback Plan:

Differential Revision: D78308133

yiming0416 commented:
@pytorchbot merge -f "landed internally, doc change only"

yiming0416 commented:
@pytorchbot merge -f "landed internally"
